launcher.py

```diff
@@ -18,9 +18,9 @@
 	from ConfigParser import ConfigParser
 
 def gen_launcher_command(settings):
-	exe_path=settings['path']
+	exe_path=get_ffxiv_path(settings)
 
-	if(settings['use_dx11'].strip() == 'True'):
+	if(settings['use_dx11'] == True):
```

Comment:
Unfortunately, this doesn't work.  It's actually undoing commit 9ccbe8465944f67e9fd6bd0a18c979d0917d7f4b :(